doc/source/tutorial/optimize.rst

```diff
@@ -1541,6 +1541,63 @@ If we need greater accuracy, typically at the expense of speed, we can solve usi
     success: True
           x: array([ 9.41025641,  5.17948718, -0.25641026,  1.64102564])  # may vary
 
+Linear sum assignment problem example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Consider planning a swimming medley relay team assignment problem.
+We have a table showing the individual medley times of five students:
+
+==========  ===========  ============  ===========  ===============================
+ Student    backstroke   breaststroke  butterfly    freestyle
+==========  ===========  ============  ===========  ===============================
+ A          43.5           47.1         48.4        38.2
+ B          45.5           42.1         49.6        36.8
+ C          43.4           39.1         42.1        43.2
+ D          46.5           44.1         44.5        41.2
+ E          46.3           47.8         50.4        37.2
+==========  ===========  ============  ===========  ===============================
+
+We need to choose best four students for each swimming style.
+This is a typical linear sum assignment problem, we can use :func:`linear_sum_assignment` to solve it.
+
+First of all, we need to setup a cost matrix.
+:func:`linear_sum_assignment` can assign each row to the best column.
+We can setup the cost matrix based on the upper table:
```

Comment:
Please explain why we need to transpose the table.

Reply:
I added. PTAL.